plugins/inputs/ethtool/README.md

```diff
@@ -14,6 +14,12 @@ on the network device and driver.
   ## List of interfaces to ignore when pulling metrics.
   # interface_exclude = ["eth1"]
 
+  ## Specifies plugin behavior regarding metrics for downed interfaces.
+  ## Available choices:
+  ##   - expose: Telegraf will expose all available metrics for downed interfaces
+  ##   - skip: Telegraf will not expose all available metrics for downed interfaces
+  # interface_down_metrics = "expose"
```

Comment:
Some bike-shedding on the option name and descriptions, otherwise this is probably good:

```suggestion
  ## Plugin behavior for downed interfaces
  ## Available choices:
  ##   - expose: collect & report metrics for down interfaces
  ##   - skip: ignore interfaces that are marked down
  # down_interfaces = "expose"
```